frontend/src/old-pages/Configure/Storage.js

```diff
@@ -494,6 +504,7 @@ function StorageInstance({index}) {
   const existingPath = STORAGE_TYPE_PROPS[storageType].mountFilesystem ? [...settingsPath, 'FileSystemId'] : [...settingsPath, 'VolumeId'];
```

Comment:
This line causes an exception when the "Add Storage" button is clicked but no storage type is selected

Reply:
Yes, I had a discussion about this with @mtfranchetto, but he was telling me that this case shouldn't happen in this component (the line above with the `"none"` default shouldn't be needed).

Reply:
This was my understanding.